tools/pyi/gen_pyi.py

```diff
@@ -449,14 +449,20 @@ def gen_pyi(declarations_path, out):
                     .format(FACTORY_PARAMS),
                     'def randint(high: _int, size: _size, *, {}) -> Tensor: ...'
                     .format(FACTORY_PARAMS)],
+        'full': ['def full(size: _size, fill_value: Number, *,'
+                 ' out: Optional[Tensor]=None, {}) -> Tensor: ...'
+                 .format(FACTORY_PARAMS),
+                 'def full(size: _size, fill_value: Number, *,'
+                 ' names: List[Union[str, None]], {}) -> Tensor: ...'
```

Comment:
I didn't review this file, just noting we don't seem to have other names overloads.

Reply:
That's true, but I think that's because the other functions specified here, like randint, can't accept a names parameter. If they can, seem like it would be a bug (or lack of a feature) in their hinting?